includes/llms.functions.core.php

```diff
@@ -483,10 +483,15 @@ function llms_form_field( $field = array(), $echo = true ) {
 			break;
 
 		case 'checkbox':
-		case 'radio':
 			$checked = ( true === $field['selected'] ) ? ' checked="checked"' : '';
 			$r .= '<input class="llms-field-input' . $field['classes'] . '" id="' . $field['id'] . '" type="' . $field['type'] . '"' . $checked . $disabled_attr . $name_attr . $required_attr . $value_attr . $field['style'] . '>';
 			$r .= $label;
+
```

Comment:
Having no break between checkbox and radio means that checkboxes end up having duplicate labels...

Enabling a T&C page via LifterLMS settings will show this issue:

![purchase_ _llms-72_dev](https://user-images.githubusercontent.com/1290739/37424675-4f8c1ea8-277e-11e8-826b-588ac67daf53.jpg)

Reply:
argh I forgot that